.github/ISSUE_TEMPLATE/GITHUB_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+---
+name: '❔ GitHub Request'
+about: 'Request update to permission / management / settings on opensearch-project GitHub organization and its repositories'
+title: '[GitHub Request]'
+labels: 'github-request'
+assignees: ''
+---
+
+### What is the type of request?  
+* _User Permissions (e.g., organization membership, repository access)_  
+* _Repository Management (e.g., creation, deletion, settings updates)_  
+* _GitHub Actions (e.g., higher-spec runners, GPU-enabled runners, troubleshooting)_  
+* _Others_
+
+### Where does this request apply?  
+* _Repository_  
+* _Organization_
+
+### Details of your request  
+_Describe the actions you are requesting (e.g., GitHub Actions in the`<repository> take too long to allocate runners)._
+
+### If this is about a repository, did you contact the maintainer first?  
+_If yes, explain why you are opening this request after consulting with the maintainer._
```

Comment:
What are you looking to get from this question>

Reply:
I am trying to check if that is something maintainers could do, but didnt.
Or does the user contact maintainer for repo level changes before contact the admins through this template.

Reply:
Removed this section.